### ISSUE TITLE
File utils.mk not found executing `make -f vendor/norsys/rothenberg/Makefile install`

### ISSUE THREAD

**Issue by c-salinas** [norsys/rothenberg#23]
After I installed rothenberg on a symfony 3.4 project, executing `docker pull norsys/rothenberg && wget -O - https://github.com/norsys/rothenberg/raw/master/install.sh | sh` without any problem,

I execute the following : 
`make -f vendor/norsys/rothenberg/Makefile install WITH_DEBUG=yes`

which outputs this :
`vendor/norsys/rothenberg/Makefile:13: resources/env/utils.mk: Aucun fichier ou dossier de ce type
make: *** Aucune règle pour fabriquer la cible « resources/env/utils.mk ». Arrêt.`

But if I do :
`ls -l vendor/norsys/rothenberg/resources/env/`

I get the following output, which confirm the presence of this file :
```
-rw-r--r-- 1 csalinas root 2919 nov.  22 11:29 app.mk
drwxr-xr-x 2 csalinas root 4096 nov.  22 11:29 bin
-rw-r--r-- 1 csalinas root  602 nov.  22 11:29 bundle.mk
-rw-r--r-- 1 csalinas root 6727 nov.  22 11:29 common.mk
-rw-r--r-- 1 csalinas root  281 nov.  22 11:29 docker-compose.rothenberg.yml
drwxr-xr-x 2 csalinas root 4096 nov.  22 11:29 nginx
drwxr-xr-x 2 csalinas root 4096 nov.  22 11:29 php
-rw-r--r-- 1 csalinas root 1019 nov.  22 11:29 utils.mk
```

chmod it doesn't change anything.

- Ubuntu 18.04.1 LTS
- docker version 18.06.1-ce, build e68fc7a
- docker-compose version 1.17.1, build unknown
- GNU Make 4.1
- output of `docker ps -a` : empty.
- output of `docker network ls` : 
NETWORK ID          NAME                DRIVER              SCOPE
67a5ec51a4cf        bridge              bridge              local
d67e3e95247a        host                host                local
fc2e7076bd1e        none                null                local

**Comment by mageekguy** [comment]
My feeling is that the command `make -f vendor/norsys/rothenberg/Makefile install …` is now useless and must be replace by `make install` in the README, but a check is mandatory before do that, because i'm not sure.

**Comment by mageekguy** [comment]
Fixed by 937d5f0.
